sunpy/timeseries/sources/rhessi.py

```diff
@@ -76,18 +76,26 @@ def peek(self, title="RHESSI Observing Summary Count Rate", **kwargs):
         ----------
         title : `str`
             The title of the plot.
+        colors : list
+            A list of colors for each line in the plot.
         **kwargs : `dict`
             Additional plot keyword arguments that are handed to `axes.plot` functions
         """
         # Check we have a timeseries valid for plotting
         self._validate_data_for_plotting()
 
+        # These are a matplotlib version of the default RHESSI color cycle
+        default_colors = ('black', 'tab:pink', 'tab:green', 'tab:cyan',
+                          'tab:olive', 'tab:red', 'tab:blue', 'tab:orange',
+                          'tab:brown')
+        colors = kwargs.pop('colors', default_colors)
+
         figure = plt.figure()
         axes = plt.gca()
 
-        for item, frame in self.to_dataframe().items():
-            axes.plot_date(self.to_dataframe().index, frame.values, '-',
-                           label=item, **kwargs)
+        for color, (item, frame) in zip(colors, self.to_dataframe().items()):
```

Comment:
What's the faliure mode look like here if colors is the wrong length?

Reply:
If colors is too long it will just use the first *n* colors, where *n* is the number of lines in the normal plot.
If colors is too short it will just plot the first *m* lines, where *m* is the number of colors. Should probably change that, but I'm not sure what to? Maybe cycling through the colors provided?

Reply:
yeah cycling sounds good.

Reply:
Once this change is done, we can merge this?

Reply:
I think so - can someone leave me a "needs changes" review to block the PR and remind me to do this?